src/main/java/com/github/havardh/javaflow/phases/parser/java/ClassVisitor.java

```diff
@@ -54,10 +58,31 @@ public void visit(ImportDeclaration n, ClassBuilder builder) {
   /** {@inheritDoc} */
   @Override
   public void visit(ClassOrInterfaceDeclaration n, ClassBuilder builder) {
-    super.visit(n, builder);
+    classNames.addLast(n.getName());
```

Comment:
Nice way of handling this 👍. I usually end up with passing this through the call stack by appending to an extra parameter, this is way nicer.